GitHub.Collectors/Config/GitHubConfigManager.cs

```diff
@@ -14,11 +14,13 @@ namespace Microsoft.CloudMine.Core.Collectors.Config
     public class GitHubConfigManager : ConfigManager
```

Comment:
If this is going to be part of all collectors (GH + ADO), should we have this in ConfigManager instead of here?